src/checker.py

```diff
@@ -155,3 +164,13 @@ def get_outdated_external_data(self):
             for data in self._external_data
             if data.state == ExternalData.State.BROKEN or data.new_version
         ]
+
+    def update_manifests(self):
+        """Updates references to external data in manifests."""
+        changed = any(data.update() for data in self._external_data)
+        if changed:
```

Comment:
This needs to be per-file; otherwise, unchanged files get spuriously reindented. (Found while testing on https://github.com/flathub/com.adobe.Flash-Player-Projector)